### PR TITLE
dhaulagir/multi-sprite

### PR DESCRIPTION
**Pull request by bguiz**
More commits after this patch: https://github.com/bguiz/ember-sprite/pull/13

### CODE REVIEW

**Comment by Dhaulagiri** [comment]
Nice! I like that reorganization better myself.

File: README.md [hunk 1]
```diff
@@ -33,31 +33,64 @@ All the configuration options are exactly the same as those in
...
-            prefix: 'icon-',
-            spritePath: '/assets/sprites.png',
-            pixelRatio: 2,
-          },
-          layoutOptions: {
-            padding: 2,
-          }
-        }
-    });
+you can configure your app, in `Brocfile.js`, like so:
```
Comment (Dhaulagiri) [inline README.md]: Upper case Y to start this line?

Comment (Dhaulagiri) [inline README.md]: Never mind, this isn't a new line

**Comment by Dhaulagiri** [comment]
I'm not sure how feasible it is, but a future enhancement to this that would be cool would be to build out the dummy app inside this addon with an example setup.

**Comment by bguiz** [comment]
This went into `ember-sprite@0.2.1` on npm

**Comment by Dhaulagiri** [comment]
Nice, I will pull it into our app and report any issues.

**Comment by bguiz** [comment]
On 8 July 2015 at 14:17, Brian Runnells <notifications@github.com> wrote:
>
> I'm not sure how feasible it is, but a future enhancement to this that would be cool would be to build out the dummy app inside this addon with an example setup.

Yeah, that sounds like a good idea.

I think the first step towards that would be add some unit/ integration tests.

W: http://bguiz.com

**Comment by bguiz** [comment]
On 8 July 2015 at 14:19, Brian Runnells <notifications@github.com> wrote:
>
> Nice, I will pull it into our app and report any issues.

Sounds good.

If you have a sample app where you're using retina images, I'd
apreciate if you could update a couple of things in `README.md`:

- the `new EmberApp()` section to add a second sprite config object in the array
- the "Full example" section to `curl` both {non-,}retina images into
the sprite folders

... as this is something that I have not yet explicitly tested.

W: http://bguiz.com

**Comment by Dhaulagiri** [comment]
Tests would be good (I would have added some if I knew how to test this type of addon).  We are indeed using retina icons, but our setup is somewhat specific to our app.  I'll see if I can do something generic for the example in the readme.

**Pull request merged**